### ISSUE TITLE
Does not rotate with mouse wheel in PrePlan view

### ISSUE THREAD

**Issue by Wuwukus** [skiselkov/BetterPushbackC#457]
Aircraft position does not rotate with mouse wheel in PrePlan view.
X-Plane 11.55, BetterPushback 0.52 (or 0.48).

**Comment by DiamondPG1** [comment]
Can we get a log?

**Comment by Wuwukus** [comment]
> Can we get a log?

X-Plane 11 log file?
[Log.txt](https://github.com/skiselkov/BetterPushbackC/files/10469314/Log.txt)

**Comment by DiamondPG** [comment]
Yes thank you, there is an error with a sound file related to the Robin DR401. Sound errors like this can cause issues BP errors. This was the only error I could find. My suggestion is to remove the aircraft and try again. If the error persists give me another log. This one didn't show any BP log errors.

The error is below:

0:02:02.064 E/SOUN: snd error in file: Aircraft/Robin DR401/fmod/DR401_CDI155.snd line 255

Let me know what happens.

**Comment by Wuwukus** [comment]
> Yes thank you, there is an error with a sound file related to the Robin DR401. Sound errors like this can cause issues BP errors. This was the only error I could find. My suggestion is to remove the aircraft and try again. If the error persists give me another log. This one didn't show any BP log errors.
> 
> The error is below:
> 
> 0:02:02.064 E/SOUN: snd error in file: Aircraft/Robin DR401/fmod/DR401_CDI155.snd line 255
> 
> Let me know what happens.
I removed the Robin 
[Log.txt](https://github.com/skiselkov/BetterPushbackC/files/10472884/Log.txt)
DR401 but it still doesn't rotate.
I found an error in the log file:
2023-01-21 21:36:00 BetterPushback[acf_outline.c:217]: Error parsing acf file: property _wing/15/_Croot not found

**Comment by RogerK888** [comment]
Any update on this? I cleared all preferences by deleting the preferences folder. Since then, BP does not work.
Cannot rotate the aircraft with the mouse as above, and when hitting enter, the screen goes away, and there is no voice from the BP crew.

**Comment by Wuwukus** [comment]
> Any update on this? I cleared all preferences by deleting the preferences folder. Since then, BP does not work.
> Cannot rotate the aircraft with the mouse as above, and when hitting enter, the screen goes away, and there is no voice from the BP crew.

No, no news.  Only mouse rotation does not work for me, the rest works fine.  The plane is pushed out only in a straight line.

**Comment by RogerK888** [comment]
> 

After messing around for hours last night. I managed to somehow get it all working again by deleting the preferences folder, loading up the intro aircraft and airport, changing it to a 737, and miraculously it started working full function. 
This proves it is not to do with any other aircraft or addons. I still don't know the exact reason it's working again, but to be honest, I am glad it is.

**Comment by Wuwukus** [comment]
> After messing around for hours last night. I managed to somehow get it all working again by deleting the preferences folder, loading up the intro aircraft and airport, changing it to a 737, and miraculously it started working full function.
> This proves it is not to do with any other aircraft or addons. I still don't know the exact reason it's working again, but to be honest, I am glad it is.

What is the settings folder?

**Comment by RogerK888** [comment]
> > After messing around for hours last night. I managed to somehow get it all working again by deleting the preferences folder, loading up the intro aircraft and airport, changing it to a 737, and miraculously it started working full function.
> > This proves it is not to do with any other aircraft or addons. I still don't know the exact reason it's working again, but to be honest, I am glad it is.
> 
> What is the settings folder?

OK, I think I now have this worked out... I'm on a Mac, so not sure if the same issue for everyone on PC.
If you are running 2 displays, you need to be running X Plane on your MAIN display. 
When I did the preferences folder reset by deleting it, it worked because X Plane defaulted to my main display. When I changed settings to run X Plane on my larger secondary display, BP stopped working. When I changed it back to the main display, it started working again. I can now reproduce this error.
Solution: 
1) Run X Plane on your main display
2) Change your MacOS preferences to make the display you want to run on the main display. You do this from:
System Preferences > Displays
Right-click the appropriate display and select it as the main display.
Start X Plane, and BP should be working like it should.

Hope this helps.

**Comment by Wuwukus** [comment]
> > > After messing around for hours last night. I managed to somehow get it all working again by deleting the preferences folder, loading up the intro aircraft and airport, changing it to a 737, and miraculously it started working full function.
> > > This proves it is not to do with any other aircraft or addons. I still don't know the exact reason it's working again, but to be honest, I am glad it is.
> > 
> > 
> > What is the settings folder?
> 
> OK, I think I now have this worked out... I'm on a Mac, so not sure if the same issue for everyone on PC. If you are running 2 displays, you need to be running X Plane on your MAIN display. When I did the preferences folder reset by deleting it, it worked because X Plane defaulted to my main display. When I changed settings to run X Plane on my larger secondary display, BP stopped working. When I changed it back to the main display, it started working again. I can now reproduce this error. Solution:
> 
> 1. Run X Plane on your main display
> 2. Change your MacOS preferences to make the display you want to run on the main display. You do this from:
>    System Preferences > Displays
>    Right-click the appropriate display and select it as the main display.
>    Start X Plane, and BP should be working like it should.
> 
> Hope this helps.

I'm play on windows PC and a single display.

**Comment by RogerK888** [comment]
> > > > After messing around for hours last night. I managed to somehow get it all working again by deleting the preferences folder, loading up the intro aircraft and airport, changing it to a 737, and miraculously it started working full function.
> > > > This proves it is not to do with any other aircraft or addons. I still don't know the exact reason it's working again, but to be honest, I am glad it is.
> > > 
> > > 
> > > What is the settings folder?
> > 
> > 
> > OK, I think I now have this worked out... I'm on a Mac, so not sure if the same issue for everyone on PC. If you are running 2 displays, you need to be running X Plane on your MAIN display. When I did the preferences folder reset by deleting it, it worked because X Plane defaulted to my main display. When I changed settings to run X Plane on my larger secondary display, BP stopped working. When I changed it back to the main display, it started working again. I can now reproduce this error. Solution:
> > 
> > 1. Run X Plane on your main display
> > 2. Change your MacOS preferences to make the display you want to run on the main display. You do this from:
> >    System Preferences > Displays
> >    Right-click the appropriate display and select it as the main display.
> >    Start X Plane, and BP should be working like it should.
> > 
> > Hope this helps.
> 
> I'm play on windows PC and a single display.

Try deleting the preferences folder in X Plane > Output > Preferences.

Back it up first. The sim will reset to all defaults. You will still have all your aircraft and plugins. See if that works for you.

**Comment by Wuwukus** [comment]
> > > > > After messing around for hours last night. I managed to somehow get it all working again by deleting the preferences folder, loading up the intro aircraft and airport, changing it to a 737, and miraculously it started working full function.
> > > > > This proves it is not to do with any other aircraft or addons. I still don't know the exact reason it's working again, but to be honest, I am glad it is.
> > > > 
> > > > 
> > > > What is the settings folder?
> > > 
> > > 
> > > OK, I think I now have this worked out... I'm on a Mac, so not sure if the same issue for everyone on PC. If you are running 2 displays, you need to be running X Plane on your MAIN display. When I did the preferences folder reset by deleting it, it worked because X Plane defaulted to my main display. When I changed settings to run X Plane on my larger secondary display, BP stopped working. When I changed it back to the main display, it started working again. I can now reproduce this error. Solution:
> > > 
> > > 1. Run X Plane on your main display
> > > 2. Change your MacOS preferences to make the display you want to run on the main display. You do this from:
> > >    System Preferences > Displays
> > >    Right-click the appropriate display and select it as the main display.
> > >    Start X Plane, and BP should be working like it should.
> > > 
> > > Hope this helps.
> > 
> > 
> > I'm play on windows PC and a single display.
> 
> Try deleting the preferences folder in X Plane > Output > Preferences.
> 
> Back it up first. The sim will reset to all defaults. You will still have all your aircraft and plugins. See if that works for you.

Deleted the preferences folder and everything worked, thanks.
